static/css/training.css

```
@@ -7,42 +7,121 @@
   clear: both;
   padding: 0px;
   margin-bottom: 2em;
+  width: 100%;
 }
 
-.col {
+section.call-to-action {
+  color: #ffffff;
+  background-color: #3371e3;
+}
+
+section.call-to-action .main-section {
+    max-width: initial;
+}
+
+section.call-to-action .main-section > div.call-to-action {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  flex-basis: auto;
+  justify-content: center;
+  align-items: center;
+  margin: initial;
+  padding-top: 5rem;
+  padding-bottom: 5rem;
+}
+
+section.call-to-action .main-section > div.call-to-action > div {
+  order: 1;
+  padding: 20px;
+}
+
+section.call-to-action .main-section .cta-text {
+  text-align: center;
+  flex: 1 1 auto;
+  max-width: 50vw;
+}
+
+/* if max() and min() are available, use them */
+section.call-to-action .main-section .cta-text {
+  min-width: min(20em, 50vw);
+  max-width: min(1000px, 50vw);
+}
+
+section.call-to-action .main-section .cta-image.cta-image-before {
+  order: 0
+}
+
+section.call-to-action .main-section .cta-image.cta-image-after {
+  order: 2
+}
+
+section.call-to-action .main-section .cta-image > img {
   display: block;
-  float:left;
+  width: 150px;
+  margin: auto;
+}
+
+.col {
+  display: flex;
+  flex-direction: row;
+  float: left;
   margin: 1% 0 1% 1.6%;
-  background-color: #f9f9f9;
 }
+
 .col:first-child { margin-left: 0; }
 
 .col-container {
-  display: table; /* Make the container element behave like a table */
+  display: flex; /* Make the container element behave like a table */
+  flex-direction: row;
   width: 100%; /* Set full-width to expand the whole page */
   padding-bottom: 30px;
 }
 
 .col-nav {
-  display: table-cell; /* Make elements inside the container behave like table cells */
+  display: flex;
+  flex-grow: 1;
   width: 18%;
   background-color: #f9f9f9;
   padding: 20px;
   border: 5px solid white;
 }
 
 
+@media only screen and (max-width: 800px) {
```

With the bigger images, I recommend:
```suggestion
 @media only screen and (max-width: 760px) {
```

This avoids a bump when the large images wrap on the page. There might be a better use of CSS to avoid that; I don't know it.